bors.toml

```diff
@@ -1,5 +1,5 @@
 status = [
-  "Taskcluster (push)"
+  "Taskcluster/complete-push"
```

Comment:
How does it look to you @pocmo ? I'm unsure how to test this.

Reply:
I just commented with `bors try` to test it :)

Reply:
Oh, wait, this has to go to master first of course.

Reply:
Anyhow, I see this error again:
https://github.com/mozilla-mobile/android-components/commit/80b89637007d9ec9284a418e86caa36eb0d3018a#comments

Reply:
Hi @owlishDeveloper! I'm not too sure to understand the error @pocmo pointed out. It seems the email address isn't considered valid. I thought it was because of 

https://github.com/mozilla-mobile/android-components/blob/44eac4b1887d060ca2589b9a4b864bcea57f2460/bors.toml#L13

=> it contains a + in it, which I know breaks some validators. Although, I see [Taskcluster relies on](https://github.com/taskcluster/taskcluster/blob/b3b815e45b0d59295bc20642f518a75614893fbd/libraries/validate/src/index.js#L8) a 3rd party library which seems to [support it](https://github.com/nwoltman/ajv/blob/8578816435590a9b9f0fc167e677e2ac82b142c0/lib/compile/formats.js#L46). So I'm wondering what email address we're actually passing down.

Do you have access to logs that may help us out?

Reply:
@JohanLorenzo We had this before: https://github.com/mozilla-mobile/android-components/pull/4097/files - In that situation it wasn't the email address of the committer but the one we assembled from the github event (see linked PR diff). Is this the same here?

Reply:
Ooooh, thanks for the info! Then, the fix is easy 🙂

Reply:
Ooooh, thanks for the info! Then, the fix is easy 🙂

Done!